spec/message.abnf

```diff
@@ -36,11 +36,11 @@ quoted-char = %x0-5B         ; omit \
             / %x7D-D7FF      ; omit surrogates
             / %xE000-10FFFF
 
-; based on https://www.w3.org/TR/xml/#NT-Nmtoken,
-; but cannot start with U+002D HYPHEN-MINUS or U+003A COLON ":"
-unquoted = unquoted-start *name-char
+; based on https://www.w3.org/TR/xml/#NT-Nmtoken
```

Comment:
this is increasingly less true. Perhaps "similar to https://...."?

Reply:
```suggestion
; similar to https://www.w3.org/TR/xml/#NT-Nmtoken
```